src/uu/mv/src/mv.rs

```diff
@@ -631,6 +634,10 @@ fn rename_with_fallback(
                     None
                 };
 
+            #[cfg(all(unix, not(target_os = "macos")))]
+            let xattrs =
+                fsxattr::retrieve_xattrs(from).unwrap_or_else(|_| std::collections::HashMap::new());
```

Comment:
Is there some reason why `xattrs` is set here and not after the following `if`-block? Otherwise you could put this snippet, and the one on line 652, into one block and save one `cfg` (and show that they belong together).

Reply:
yeah, I would have to duplicate the declaration as the xattr detection needs to happen with and without the progress bar

and you can't do at the same time because "from" gets removed/moved :)

so, to rephrase
1) get the attributes on the file
2) move the file
3) set the attributes on the "new" file